crates/web-sys/src/features/gen_CanvasPattern.rs

```diff
@@ -19,5 +19,5 @@ extern "C" {
     #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/CanvasPattern/setTransform)"]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `CanvasPattern`, `SvgMatrix`*"]
-    pub fn set_transform(this: &CanvasPattern, matrix: &SvgMatrix);
+    pub fn set_transform_with_default_dom_matrix_2d_init(this: &CanvasPattern, matrix: &SvgMatrix);
```

Comment:
This is a breaking change (and not really a desirable one, either).

I can see what happened here: you used `FIXED_INTERFACES` to prevent `CanvasRenderingContext2d::set_transform` being changed to mean the `DomMatrix` overload with its default value, but that affected this method too since it doesn't distinguish between methods on different types.

I think `FIXED_INTERFACES` will have to be extended to allow specifying both the type name and method name to fix this.